CoreGraphFile/src/au/gov/asd/tac/constellation/graph/file/GraphObjectUtilities.java

```diff
@@ -29,11 +36,28 @@
  */
 public class GraphObjectUtilities {
 
+    private static final GraphObjectUtilities graphObjectUtilities = new GraphObjectUtilities();
```

Comment:
Based on what I could see from the usage, I think it would be better to make relevant functions and variables being called static rather than creating a static instance of the class within itself (doing a bit of testing, it seems to work the same).

Reply:
Changed the other methods to static so this isn't needed.